packages/server/graphql/mutations/createReflection.ts

```diff
@@ -57,14 +58,18 @@ export default {
 
     // RESOLUTION
     const plaintextContent = extractTextFromDraftString(normalizedContent)
-    const entities = await getReflectionEntities(plaintextContent)
+    const [entities, sentimentScore] = await Promise.all([
+      getReflectionEntities(plaintextContent),
+      getReflectionSentimentScore(plaintextContent)
```

Comment:
+1 did you try combining both entity and sentiment analyzis? https://cloud.google.com/natural-language/docs/analyzing-entity-sentiment
It doesn't make a cost difference, but would save one roundtrip.

Reply:
I think that API returns entities + **entity-level** sentiment score, whereas here we'd like a sentiment score for the entire document. The aggregations of the score from each entity doesn't necessarily represent the score for the entire document, e.g.

<img width="950" alt="Screenshot 2023-03-13 at 11 12 46" src="https://user-images.githubusercontent.com/1879975/224792113-feb78ca8-abfe-415f-b871-5712d5acad9f.png">